packages/ui/src/themes/html/RangeSlider.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { forwardRef } from "react";
+import { RangeSliderProps } from "../../theming/ThemeType";
+import * as styles from "./RangeSlider.module.css";
+
+export const RangeSlider = forwardRef<HTMLInputElement, RangeSliderProps>(
+	({ children, ...props }, ref) => {
+		const { min, max, step, list } = props;
+
+		return (
+			<>
+				<input
+					type="range"
+					min={min}
+					max={max}
+					step={step}
+					list={list}
+				/>
+				<datalist id="values" className={styles.default.dataList}>
```

Comment:
You can generate the name of the field of the id (props.name + "_list") ? so list does not need to be a prop of this component